pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -92,23 +91,7 @@ var _ = Describe("ImageUpload", func() {
 		defer archiveFile.Close()
 		archiveFilePath = archiveFile.Name()
 
-		// Get the image into a tar archive
-		w := tar.NewWriter(archiveFile)
-		defer w.Close()
-
-		srcFile, err := os.Open(imagePath)
-		Expect(err).ToNot(HaveOccurred())
-		defer srcFile.Close()
-
-		srcFileInfo, err := srcFile.Stat()
-		Expect(err).ToNot(HaveOccurred())
-		hdr, err := tar.FileInfoHeader(srcFileInfo, "")
-		Expect(err).ToNot(HaveOccurred())
-
-		err = w.WriteHeader(hdr)
-		Expect(err).ToNot(HaveOccurred())
-		_, err = io.Copy(w, srcFile)
-		Expect(err).ToNot(HaveOccurred())
+		libstorage.ArchiveToFile(archiveFile, imagePath)
```

Comment:
Here, I'm a bit confused, haven't you removed this function in 93ffc8a692e52920db846b7e23980ff454f86d0f ?

Reply:
Maybe I should remove this commit completely because I then moved `ArchiveToFile` and  `ChangeImgFilePermissionsToNonQEMU` to a new file in a different commit.